cmd/result.go

```diff
@@ -59,6 +59,11 @@ func createFields(res Result, occ Occurrence) (fields log.Fields) {
 		fields[k] = v
 	}
 	fields["Container"] = occ.container
+
+	if occ.id == ErrorRunAsNonRootFalse {
+		fields["Pod"] = occ.podHost
```

Comment:
Pod should only be set if there is actually something in the podHost variable because otherwise it shows up but is empty:
```
ERRO[0000] RunAsNonRoot is set to false (root user allowed), please set to true!  Container=fakeContainerRANR KubeType=pod Name=run_as_non_root_psc_false_csc_false Namespace=fakeDeploymentRANR Pod=
```

Reply:
this has been fixed. Great Suggestion!